application/actions/dependents.go

```diff
@@ -124,6 +126,41 @@ func dependentsUpdate(c buffalo.Context) error {
 	return c.Render(http.StatusOK, r.JSON(output))
 }
 
+// swagger:operation DELETE /policy-dependents/{id} PolicyDependents PolicyDependentsDelete
+//
+// PolicyItemsDelete
+//
+// Delete a policy dependent if it has no related policy items.
+//
+// ---
+// parameters:
+//   - name: id
+//     in: path
+//     required: true
+//     description: policy dependent ID
+// responses:
+//   '204':
+//     description: OK but no content in response
+func dependentsDelete(c buffalo.Context) error {
+	tx := models.Tx(c)
+	dependent := getReferencedDependentFromCtx(c)
+
+	relatedItemNames := dependent.RelatedItemNames(tx)
+	if len(relatedItemNames) == 0 {
```

Comment:
In my opinion, it would read better to handle the error first.
```suggestion
	if len(relatedItemNames) >  0 {
```